app/main/views/questionnaire.py

```diff
@@ -240,33 +218,31 @@ def post_household_composition(eq_id, form_type, collection_id, group_id):
     questionnaire_manager = get_questionnaire_manager(g.schema, g.schema_json)
     answer_store = get_answer_store(current_user)
 
-    this_block = {
-        'block_id': 'household-composition',
-        'group_id': group_id,
-        'group_instance': 0,
-    }
+    this_location = Location(group_id, 0, 'household-composition')
```

Comment:
It's a preference thing but I think it's beneficial to keep Location(...) one line as long as it fits but named params make sense where applicable